pkg/trace/api/api_test.go

```diff
@@ -368,6 +368,31 @@ func TestReceiverDecodingError(t *testing.T) {
 	})
 }
 
+func TestReceiverUnexpectedEOF(t *testing.T) {
+	assert := assert.New(t)
+	conf := newTestReceiverConfig()
+	r := newTestReceiverFromConfig(conf)
+	server := httptest.NewServer(http.HandlerFunc(r.handleWithVersion(v05, r.handleTraces)))
+	// we get to read the header and the entire dictionary, but the request claims to be much larger
+	data := []byte{0x92, 0x91, 0xA5, 'a', 'b', 'c', 'd', 'e'}
+	client := &http.Client{}
```

Comment:
```suggestion
	var client http.Client
```
Your version is also correct, but this is the general convention.

Reply:
Well, this isn't the convention used elsewhere in this file

Reply:
This is the way you wrote it yourself elsewhere.
<img width="738" alt="Screenshot 2020-09-07 at 13 22 43" src="https://user-images.githubusercontent.com/16439049/92387340-4142ff00-f10d-11ea-9b8b-5b7c106ee67f.png">

Reply:
Doesn't matter. This is what we want moving forward.

Reply:
Richard, I didn't write the entire codebase, I copied it from its old location at https://github.com/DataDog/datadog-trace-agent so I show up on the entire code (minus recent changes since the copy). Can we focus on the actual work and moving the PR forward?